ios/keyman/Keyman/Keyman/Classes/SetUpViewController/SetUpViewController.swift

```diff
@@ -7,12 +7,12 @@
 //
 
 import KeymanEngine
-import UIKit
+import WebKit
 import Reachability
 
 // TODO: Refactor common functionality from InfoViewController
-class SetUpViewController: UIViewController, UIWebViewDelegate {
-  @IBOutlet var webView: UIWebView!
+class SetUpViewController: UIViewController, WKNavigationDelegate {
```

Comment:
Note:  it is very rare, at this point, for this class to ever actually be utilized... if at all.

It is only ever referenced by:

https://github.com/keymanapp/keyman/blob/f6841333a251769eaa5cd25b10b48d63145c4db8/ios/keyman/Keyman/Keyman/Classes/GetStartedViewController/GetStartedViewController.swift#L155-L161

But https://developer.apple.com/documentation/uikit/uiapplication/1623042-opensettingsurlstring indicates that the function likely always exists and returns something valid.

That said... we do still have to have the `if let _` construct because the initialization of the `URL` technically returns an optional, and the update matches that for the testable InfoViewController, so... it shouldn't pose any actual issues if we leave it in for now, at least.  But _maybe_ it's actually safe to eliminate it?